CHANGELOG.md

```diff
@@ -5,6 +5,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
```

Comment:
I'm not happy with this "removed"/"changed"/"fixed"/etc. as it fails to clearly communicate where we have breaking changes, versus where we don't.

If you want to keep it, can you please make sure that all breaking changes are properly tagged as such?
As in (examples) ESP IDF v4.3 support, the ledc API changes and possibly others.

Reply:
Well this format of using removed/changed/fixed etc is directly what is suggested on the link we have in the first line of the changelog itself `Keep a Changelog` and is used like this in most Changelogs i am aware of. For example like in [esp-idf](https://github.com/espressif/esp-idf/releases/tag/v5.2.2), [probe-rs](https://github.com/probe-rs/probe-rs/blob/master/CHANGELOG.md), [tokio](https://github.com/tokio-rs/tokio/blob/master/tokio/CHANGELOG.md) etc. 

a "Remove" is always a breaking change. And in general something that is in "Changed" is also breaking. ( But i may have failed here to categorize everything into its correct category, so if you think i put something into a wrong bucket tell please. 

But if you want i can add either a Breaking Tag before the changes or the change category itself.

Reply:
Top priority for me is to be crystal clear if something is a breaking change or not. If you can do it with the format you have adopted or some variation of it - fine. If you can't, just switch to the old format, where we did not care to have separate "added/removed/changed" sections but just communicated this piece of info as part of each item.

What is not acceptable for me is to adopt the format "everybody uses" and thus lose the one single most important thing we have to clearly communicate to users (and track ourselves) - how many, and what breaking changes we introduce with each new release.

As we are no longer in the "we can break everything, not so important" operation.